src/flow.js

```diff
@@ -0,0 +1,8 @@
+import PostToConversation from './PostToConversation'
```

Comment:
```suggestion
import FlowPostToConversation from './views/FlowPostToConversation'
```
Can this be moved to`src/views/FlowPostToConversation.vue`?

Reply:
sure